README.md

```diff
@@ -189,21 +189,21 @@ spec:
 If you are not using GKE and you don't have StorageClass for dynamic volume provisioning at your cluster, you have to create pvc and pv
 
 ```
-kubectl create -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/tfevent-volume/tfevent-pvc.yaml
+kubectl create -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1alpha1/tfevent-volume/tfevent-pvc.yaml
 
-kubectl create -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/tfevent-volume/tfevent-pv.yaml
+kubectl create -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1alpha1/tfevent-volume/tfevent-pv.yaml
 ```
 
 This is example for tfjob operator
 
 ```
-kubectl create -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/tfjob-example.yaml
+kubectl create -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1alpha1/tfjob-example.yaml
 ```
 
 This is example for pytorch operator
 
 ```
-kubectl create -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/pytorchjob-example.yaml
+kubectl create -f https://raw.githubusercontent.com/kubeflow/katib/master/examples/v1alpha1/pytorchjob-example.yaml
 ```
 
 You can check status of StudyJob
```

Comment:
@alexandraj777 Thank you for your PR! 
Should we also update the link here: https://github.com/kubeflow/katib/pull/452/files#diff-04c6e90faac2675aa89e2176d2eec7d8R161
and here: https://github.com/kubeflow/katib/pull/452/files#diff-04c6e90faac2675aa89e2176d2eec7d8R362  ?